float64 speed_fl 
float64 speed_rl 
float64 speed_fr 
float64 speed_rr
float64 v_x
float64 ome_z
